README.md

```diff
@@ -200,7 +200,7 @@ npx knex seed:make seed_name
 ```
 or its make counterpart, from the root directory
 ```shell
-SEED=<your_seed> && make seed
+SEED=<your_seed> && make seed-make
```

Comment:
we're missing an "export" in front of this, i.e.
```
export SEED=<your_seed> && make seed-make
```

Reply:
Thanks! Done